.github/workflows/flutter.yml

```diff
@@ -18,3 +18,10 @@ jobs:
     - run: flutter pub get
     - run: flutter analyze
     - run: flutter test
+    - run: flutter build apk --split-per-abi
+    - name: Create a Release APK
+      uses: ncipollo/release-action@v1
```

Comment:
We could add `if: github.ref == 'refs/heads/master'` here, so we always run `flutter build` for branches but only create an actual release for master :smile: